pkg/adapter/nimbus-kyverno/manager/manager.go

```diff
@@ -41,7 +41,7 @@ func Run(ctx context.Context) {
 
 	npCh := make(chan common.Request)
 	deletedNpCh := make(chan common.Request)
-	go globalwatcher.WatchNimbusPolicies(ctx, npCh, deletedNpCh)
+	go globalwatcher.WatchNimbusPolicies(ctx, npCh, deletedNpCh, "SecurityIntentBinding")
```

Comment:
This only watches SIB CR, but why does the KubeArmor adapter watch for both SIB and CSIB CRs?

Reply:
Please note that the function is still watching only for Nimbus Policies. The parameters passed indicate the owners of a given Nimbus Policy. 

kyverno adapter should not watch for nimbus policies whose owner is a CSIB. The reason being that kyverno adapter directly watches for the Cluster Nimbus Policy.

The design document contains the details of this.